arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -2226,34 +2226,34 @@ Result IResearchAnalyzerFeature::loadAnalyzers(
 
         return {};  // skip analyzer
       }
-
       AnalyzersRevision::Revision revision{AnalyzersRevision::MIN};
-      auto revisionSlice =
-          slice.get(arangodb::StaticStrings::AnalyzersRevision);
-      if (!revisionSlice.isNone()) {
-        revision = revisionSlice.getNumber<AnalyzersRevision::Revision>();
-      }
-      if (revision > loadingRevision) {
-        LOG_TOPIC("44a5b", DEBUG, iresearch::TOPIC)
-            << "analyzer " << name
-            << " ignored as not existed. Revision:" << revision
-            << " Current revision:" << loadingRevision;
-        return {};  // this analyzers is still not exists for our revision
-      }
-      revisionSlice =
-          slice.get(arangodb::StaticStrings::AnalyzersDeletedRevision);
-      if (!revisionSlice.isNone()) {
-        auto deletedRevision =
-            revisionSlice.getNumber<AnalyzersRevision::Revision>();
-        if (deletedRevision <= loadingRevision) {
-          LOG_TOPIC("93b34", DEBUG, iresearch::TOPIC)
+      if (ServerState::instance()->isRunningInCluster()) {
```

Comment:
```suggestion
      if (!ServerState::instance()->isSingleServer()) {
```

Reply:
We use "running in cluster" in other related places. So wanted to be consistent